docs/book/src/topics/control-plane-endpoint.md

```diff
@@ -0,0 +1,215 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Exposing the control plane endpoint](#exposing-the-control-plane-endpoint)
+  - [Octavia](#octavia)
+    - [Parameters](#parameters)
+    - [Default behaviour](#default-behaviour)
+    - [No floating IP](#no-floating-ip)
+    - [Using a pre-created floating IP](#using-a-pre-created-floating-ip)
+    - [Deletion of the floating IP](#deletion-of-the-floating-ip)
+    - [Exposing additional control plane services running on other ports](#exposing-additional-control-plane-services-running-on-other-ports)
+    - [Restricting access to the loadbalancer endpoint](#restricting-access-to-the-loadbalancer-endpoint)
+    - [Specifying an alternate primary API server port](#specifying-an-alternate-primary-api-server-port)
+    - [Alternate Octavia provider (e.g. OVN)](#alternate-octavia-provider-eg-ovn)
+    - [Bugs](#bugs)
+- [Floating IP](#floating-ip)
```

Comment:
Should `Floating IP` and `No loadbalancer` sections have the same indentation of the `Octavia` section?